pennylane/math/single_dispatch.py

```diff
@@ -133,6 +133,7 @@ def _take_autograd(tensor, indices, axis=None):
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "arctan"] = "tensorflow.math"
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "arctan2"] = "tensorflow.math"
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "diag"] = "tensorflow.linalg"
+ar.autoray._SUBMODULE_ALIASES["tensorflow", "kron"] = "tensorflow.experimental.numpy"

```

Comment:
Sorry @josh146, I went the lazy route and just added the function from here...is that ok for now? I thought that tf's experimental function is safer than anything I code cook up ;)